Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -0,0 +1,78 @@
+
+import UIKit
+
+class FormTextField: UIView {
+    
+    // MARK: - Outlets
+    //
+    
+    @IBOutlet weak private var contentView: UIView!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var inputTextField: UITextField!
+    @IBOutlet weak private var errorLabel: UILabel!
+    
+    // MARK: - Properties
+    //
+    var text: ((String)->())?
+    
+    // MARK: - init
+    //
```

Comment:
```suggestion
    // MARK: Init

```
Commented message should follow sentence case (Starts with a capital letter)